internal/packages/internal/packageimport/oci_test.go

```
@@ -46,3 +46,20 @@ func TestFromOCI_EmptyImage(t *testing.T) {
 	_, err := FromOCI(ctx, image)
 	require.EqualError(t, err, packagetypes.ErrEmptyPackage.Error())
 }
+
+func TestFromOCI_DotTarEntry(t *testing.T) {
+	t.Parallel()
+
+	labels := map[string]string{"test": "test123"}
+	image := testutil.BuildImage(t, map[string][]byte{
+		".": []byte{},
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)